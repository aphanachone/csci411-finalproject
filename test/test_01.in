0 8 0 9 4 5 2 3 7 
3 2 7 6 1 8 0 0 9 
4 5 0 7 3 2 8 0 0 
0 6 1 4 0 9 0 0 0 
8 9 4 3 5 0 6 2 1 
5 0 3 0 2 0 0 8 0 
0 0 8 5 9 0 0 0 2 
9 0 0 0 0 0 4 6 8 
7 0 0 0 6 4 3 0 5 