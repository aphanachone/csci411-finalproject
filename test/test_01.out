Solution:
1 8 6 9 4 5 2 3 7 
3 2 7 6 1 8 5 4 9 
4 5 9 7 3 2 8 1 6 
2 6 1 4 8 9 7 5 3 
8 9 4 3 5 7 6 2 1 
5 7 3 1 2 6 9 8 4 
6 4 8 5 9 3 1 7 2 
9 3 5 2 7 1 4 6 8 
7 1 2 8 6 4 3 9 5 
